### PR TITLE
docs: update OEP-50 so reflects latest documentation

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds the latest documentation to the OEP-50.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mariajgrimaldi!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mariajgrimaldi** [comment]
@felipemontoya, what do you think of these changes?

File: oeps/architectural-decisions/oep-0050-hooks-extension-framework.rst [hunk 1]
```diff
@@ -276,3 +281,6 @@ Adding an official arbiter
 
 17 April 2021 - Felipe Montoya
 Updating based on feedback from the community and edX arch team.
+
+20 July 2022 - Maria Grimaldi
```
Comment (robrap) [inline oeps/architectural-decisions/oep-0050-hooks-extension-framework.rst:285]: Nit: Change history should should be ordered most recent first. See https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#change-history

Comment (mariajgrimaldi) [reply oeps/architectural-decisions/oep-0050-hooks-extension-framework.rst:285]: I didn't know that! Thanks!

**Review by felipemontoya — APPROVED** [review]
This looks good to me. I think we should merge it.

Now, this also makes me think if we should change the status of this OEP from `provisional` -> `accepted` since the reference implementation is now created and deployed in the main branch of edx-platform.

**Comment by openedx-webhooks** [comment]
@mariajgrimaldi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/open-edx-proposals&entry.867055334=https://github.com/openedx/open-edx-proposals/pull/358&entry.1484655318=https://github.com/mariajgrimaldi&entry.752974735=2022-07-20+16:26&entry.1917517419=2022-07-28+14:13&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**